### PR TITLE
Introduce editorconfig file

### PR DESCRIPTION
**Pull request by svgradev**

### CODE REVIEW

File: modules/engines/engine-core/src/main/java/com/sap/xsk/hdb/ds/api/IXSKDataStructureModel.java [hunk 1]
```diff
@@ -13,101 +13,55 @@
...
+  /** File extension for *.hdbcalculationview files */
+  public static final String FILE_EXTENSION_HDBCALCULATION_VIEW = ".hdbcalculationview";
+  /** File extension for *.hdbprocedure files */
+  public static final String FILE_EXTENSION_HDBPROCEDURE = ".hdbprocedure";
+  /** File extension for *.hdbtablefunction files */
+  public static final String FILE_EXTENSION_HDBTABLEFUNCTION = ".hdbtablefunction";
+  /** File extension for *.hdbschema files */
+  public static final String FILE_EXTENSION_HDBSCHEMA = ".hdbschema";
+  /** File extension for *.hdi files */
+  public static final String FILE_EXTENSION_HDI = ".hdi";
```
Comment (antoanetamh) [inline modules/engines/engine-core/src/main/java/com/sap/xsk/hdb/ds/api/IXSKDataStructureModel.java:33]: This formatting looks strange.
In editconfig file there is ij_java_doc_do_not_wrap_if_one_line = false
which means that the comment should be not formatted in one line, as it is.

Comment (ThuF) [reply modules/engines/engine-core/src/main/java/com/sap/xsk/hdb/ds/api/IXSKDataStructureModel.java:33]: +1, could you revert the comments formatting to be on multiple lines?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,621 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = false
+max_line_length = 100
```
Comment (ThuF) [inline .editorconfig:7]: Please increase the max line length to 150 symbols, 100 is too little and results into multiple line statements.

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,621 @@
...
+ij_java_do_not_indent_top_level_class_members = false
+ij_java_do_not_wrap_after_single_annotation = false
+ij_java_do_while_brace_force = always
+ij_java_doc_add_blank_line_after_description = true
+ij_java_doc_add_blank_line_after_param_comments = false
+ij_java_doc_add_blank_line_after_return = false
+ij_java_doc_add_p_tag_on_empty_lines = true
+ij_java_doc_align_exception_comments = true
+ij_java_doc_align_param_comments = true
+ij_java_doc_do_not_wrap_if_one_line = false
```
Comment (ThuF) [inline .editorconfig:82]: I guess that this should be set to `true`, in order to resolve the bellow comments

**Review by ThuF — CHANGES_REQUESTED** [review]
Please fix the comments, before merging

File: .editorconfig (hunk 2 above)
Comment (ageorgievg) [reply .editorconfig:7]: We were thinking about 140. At 150 at least on my laptop it goes off the screen.

**Review by ThuF — APPROVED** [review]

**Pull request merged**